test/system/045-start.bats

```diff
@@ -27,7 +27,11 @@ load helpers
 
     run_podman wait $cid_none_implicit $cid_none_explicit $cid_on_failure
 
-    run_podman rm $cid_none_implicit $cid_none_explicit $cid_on_failure
+#   FIXME:  Just removing this flaking command until we can get it fixed.
```

Comment:
It looks like this doesn't fix the flake, so I see no reason to merge it. But if you do, could you please change to:
```
# FIXME: #10454: ...
```
Reason: that makes it possible to remove the workaround in the future